File: Python/Easy/الجمع التراكمي.py

# Creator: OmarAfet
# https://profile.satr.codes/OmarAfet/public/overview
# https://github.com/OmarAfet

def cumulative_addition(elements_array: list[int]) -> list[int]:
    length = len(elements_array)
    sum = 0
    for i in elements_array:
        sum += i
    
    return [sum, length]

File: challenges/JavaScript/1. Easy/الجمع التراكمي.js
/*

Creator: OmarAfet
https://profile.satr.codes/OmarAfet/public/overview
https://github.com/OmarAfet

*/

function cumulative_addition(elements_array) {
	let sum = 0;
	for (let i = 0; i < elements_array.length; i++) {
		sum += elements_array[i];
	}
	return [sum, elements_array.length];
}
